kafka-consumer-application/kafka/README.md

```diff
@@ -0,0 +1,160 @@
+# Kafka Consumer Application
+
+Let's build a simple application that uses a `KafkaConsumer` to read records from Kafka.
+
+## Create a Consumer
+
+There are required propertied needed to create a Kafka Consumer. At a minimum, the `Consumer` needs to know:
+* How to find the Kafka broker(s).
+* How to deserialize the key and value of events.
+* A group id value for consumer group coordination.
+
+```java annotate
+  final String bootstrapServers = "localhost:9092";
+  final String consumerGroupId = "my-group-id";
+  final Properties consumerAppProps = new Properties() {{
+        put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
+        put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        put(ConsumerConfig.GROUP_ID_CONFIG, consumerGroupId);
+        put(ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG, "500");
+        put(ConsumerConfig.MAX_POLL_RECORDS_CONFIG, "2");
```

Comment:
Is there a reason to have these configurations for this tutorial?  Both are a lot lower than the defaults.  Since this is an introductory tutorial, I say we remove these two configs.

Reply:
That was experimental, thanks for reminding me to delete it.